src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1625,112 +1616,112 @@
-  "com.minecolonies.coremod.gui.chat.recruitstory4": "is not the  tallest one, but firmly believes that even the smallest person can change the course of the future.",
-  "com.minecolonies.coremod.gui.chat.recruitstory5": "saw a village with its inhabitants going around in minecarts. It seemed like fun and maybe %s can do that here too.",
-  "com.minecolonies.coremod.gui.chat.recruitstory6": "travels around a lot, a stranger once said 'Chewie, we're home.', %s couldnt make much sense of it though",
-  "com.minecolonies.coremod.gui.chat.recruitstory7": "is rather weird, anytime you pass by %s utters 'Live long, and prosper'.",
-  "com.minecolonies.coremod.gui.chat.recruitstory8": "was a rich travelling trader until someone robbed %s of all the goods.",
-  "com.minecolonies.coremod.gui.chat.recruitstory9": "was known as a strong fighter in a far away kingdom, though for some reason %s cannot go back there.",
-  "com.minecolonies.coremod.gui.chat.recruitstory10": "has excellent masonry skills, but %s's father wanted a cow herder.",
-  "com.minecolonies.coremod.gui.chat.recruitstory11": "is a noble from a distant town. %s thinks to surely become the major of this tiny town soon",
-  "com.minecolonies.coremod.gui.chat.recruitstory12": "waves with the hand and mumbles something like tree force bee wich you",
+  "com.minecolonies.coremod.gui.chat.recruitstory1": "comes from a cold place up north. Hoping to find a warmer place, %s stopped by your tavern.",
+  "com.minecolonies.coremod.gui.chat.recruitstory2": "is not a fan of this place, but didnt see any others close by.",
+  "com.minecolonies.coremod.gui.chat.recruitstory3": "likes to wander around the world. Wandering makes them hungry and thirsty, so %s stopped by your tavern.",
+  "com.minecolonies.coremod.gui.chat.recruitstory4": "is not the biggest one, but firmly believes that even the smallest person can change the course of the future.",
+  "com.minecolonies.coremod.gui.chat.recruitstory5": "saw a village with its inhabitants going around in minecarts. It seemed like fun, and maybe %s can do that here!",
+  "com.minecolonies.coremod.gui.chat.recruitstory6": "travels around a lot. A stranger once said 'Chewie, we're home.' %s couldn't make much sense of it, though!",
+  "com.minecolonies.coremod.gui.chat.recruitstory7": "is rather weird. Anytime you pass by, %s utters 'Live long and prosper.'",
+  "com.minecolonies.coremod.gui.chat.recruitstory8": "was a rich traveling trader until someone robbed %s of all their goods.",
+  "com.minecolonies.coremod.gui.chat.recruitstory9": "was known as a strong fighter in a faraway kingdom, though for some reason %s cannot go back there.",
+  "com.minecolonies.coremod.gui.chat.recruitstory10": "has excellent masonry skills, but %s's father wanted a cow herder instead.",
+  "com.minecolonies.coremod.gui.chat.recruitstory11": "is a noble from a distant town. %s thinks they will surely become the mayor of this tiny colony soon.",
```

Comment:
They aren't becoming a mayor any time soon, considering that colonies don't have them :)

Also just putting it here because it won't let me review further down: please make "Chit-Chat" lowercase - it's bugging me.

Reply:
Emphasis on *thinks* :P